mechkit/material.py

```diff
@@ -796,17 +795,18 @@ class TransversalIsotropic(AbstractMaterial):
         - **E_l**, **E_t**, **G_lt**, **nu_lt**, **nu_tt**
         - **E_l**, **E_t**, **G_lt**, **nu_tl**, **nu_tt**
 
-    Attributes: **(** Accessible both as attributes and dict-like **)**
+    Attributes:
 
-        - **E_l**, **E_t**, **G_lt**, **G_tt**, **nu_lt**, **nu_tl**, **nu_tt**
+    - Keyword arguments
```

Comment:
There are seven attributes.. this means, the attribute list contained additional information.
I suggest we keep this explicit list and do not consider it as repetition. Is this ok?

```suggestion
       - **E_l**, **E_t**, **G_lt**, **G_tt**, **nu_lt**, **nu_tl**, **nu_tt**
```

Reply:
Ok. I copied using "keyword arguments" instead of an attribute list from the documentation of orthotropic material. Would you like me to add an attribute list there as well?